app/src/main/java/com/codesoom/assignment/domain/User.java

```diff
@@ -38,6 +39,14 @@ public void destroy() {
     }
 
     public boolean authenticate(String password) {
-        return !deleted && password.equals(this.password);
+        if (deleted) {
+            throw new AuthenticationFailException("존재 하지 않는 사용자 입니다");
```

Comment:
이런 메시지는 최종 사용자에게 도달하게 되기 때문에 어법과 띄어쓰기 등에도 신경써야 합니다.
```suggestion
            throw new AuthenticationFailException("존재하지 않는 사용자 입니다");
```

Reply:
네!